root/types.js

```diff
@@ -364,28 +364,40 @@ declare type EditorPreferencesT = {|
 
 declare type EditorT = {|
   ...EntityRoleT<'editor'>,
+  +age?: number,
+  +area: AreaT | null,
   +biography: string | null,
   +birth_date: PartialDateT | null,
   +deleted: boolean,
   +email: string,
   +email_confirmation_date: string | null,
+  +gender: GenderT,
   +gravatar: string,
+  +has_confirmed_email_address: boolean,
   +is_account_admin: boolean,
   +is_admin: boolean,
   +is_auto_editor: boolean,
   +is_banner_editor: boolean,
   +is_bot: boolean,
+  +is_charter: boolean,
   +is_editing_disabled: boolean,
   +is_limited: boolean,
   +is_location_editor: boolean,
   +is_relationship_editor: boolean,
   +is_wiki_transcluder: boolean,
+  +languages: $ReadOnlyArray<EditorLanguageT> | null,
+  +last_login_date: string | null,
   +name: string,
   +preferences: EditorPreferencesT,
   +registration_date: string,
   +website: string | null,
 |};
 
+declare type EditorLanguageT = {|
```

Comment:
It might be a good idea to define a `TO_JSON` subroutine in lib/MusicBrainz/Server/Entity/EditorLanguage.pm (rather than rely on the auto-conversion) so there aren't any surprises. :)